package.json

```diff
@@ -45,7 +46,7 @@
         "postinstall": "npx husky install",
         "prepublishOnly": "npx lerna run copy",
         "clean": "lerna run clean",
-        "build": "lerna run build",
+        "build": "lerna run build --sort",
```

Comment:
this should also not be needed, and I really do believe this is enabled by default (the option is there so you can use it as `--no-sort` to disable things)

Reply:
It seems that making scraper-tools depend on the local apify version made lerna run the build scripts in the right order, so I will remove this